wpimath/src/main/java/edu/wpi/first/math/controller/LinearQuadraticRegulator.java

```diff
@@ -124,6 +126,8 @@ public LinearQuadraticRegulator(
     m_u = new Matrix<>(new SimpleMatrix(B.getNumCols(), 1));
 
     reset();
+    MathSharedStore.getMathShared()
+        .reportUsage(MathUsageId.kController_LinearQuadraticRegulator, 0);
```

Comment:
The C++ version of this statement uses a different number. Also, does using zero here still get logged?

Reply:
Changed to 1; I had missed it when doing a pass through to update 0 to 1 for that exact reason.